cuda_bindings/docs/source/install.md

```diff
@@ -46,6 +46,12 @@ During the build process, environment variable `CUDA_HOME` or `CUDA_PATH` are us
 export CUDA_HOME=/usr/local/cuda
 ```
 
+Some other build time environment variables are as follows:
```

Comment:
Could you create a new page "environment variables", move the text there & also add run-time env vars, and link it to index.rst & cross-ref it here? This way we have env vars easily findable on table of contents. Right now you have to know what you're looking for.